packages/i18n/locales/zh/page-home.json

```diff
@@ -328,9 +328,10 @@
       "submit": "提交",
       "successMsg": "订阅成功",
       "errorMsg": "订阅失败",
+      "legalMsg": "请确认隐私条款",
```

Comment:
use privacyMsg may be a better choice? It's just a suggestion.

Reply:
Yes, that might be a better name -- but I'll keep the old one for now.